File: new/코딩도장/C언어/Multiples of 3 and 5.cpp
#include<Stdio.h>
int main()
{

	int i=0,sum=0;

	for(i=1;i<1000;i++)
	{
    	if(i%3==0||i%5==0)
    	{
    		printf("%d  ", i);
    		sum+=i;
		}
        	
	}

	printf("1000미만의 자연수에서 3,5의 배수의 총합은 : %d ",sum);

	return 0;
	
}

File: new/코딩도장/python/Multiplesof3and5.py
#집합 자료형(set 사용)
set1 = set(range(3,1000,3))
set2 = set(range(5,1000,5))
set3 = set1 | set2
result = sum(set3)
print(result)

#집합 자료형 X
sum = 0
print("sum의 초기값 %d" , sum)
for i in range(1, 1000):
    if i%3 ==0 or i% 5 ==0:
        sum+=i

print(sum)
